tests/scripts/thread-cert/Cert_5_3_02_RealmLocal.py

```diff
@@ -96,14 +96,14 @@ def test(self):
                 self.assertTrue(self.nodes[LEADER].ping(addr, size=256))
                 self.assertTrue(self.nodes[LEADER].ping(addr))
 
-        self.assertTrue(self.nodes[LEADER].ping('ff03::1', size=256))
-        self.assertTrue(self.nodes[LEADER].ping('ff03::1'))
+        self.assertTrue(self.nodes[LEADER].ping('ff03::1', num_responses=2, size=256))
+        self.assertTrue(self.nodes[LEADER].ping('ff03::1', num_responses=2))
```

Comment:
Shouldn't these be `num_responses=3` since it is pinging the Realm-Local All **Nodes** address, and there are 3 other nodes?